client/src/pages/DetailsPage/item-details.module.scss

```
@@ -0,0 +1,13 @@
+.sectionDivider {
+    border-top: 1px solid #000000;
+    margin: 20px 0;
+}
+
+.itemDetails {
+    article {
+        image {
+            width: 80vw;
```

Vergiss nicht hier eine max width zu vergeben um ein überdimensionales Bild zu vermiden.
```suggestion
            width: 80vw;
            max-width: 960px;
```

---

done!